nomos-da/kzgrs/src/rs.rs

```diff
@@ -83,7 +83,7 @@ pub fn lagrange_interpolate(points: &[Fr], roots_of_unity: &[Fr]) -> DensePolyno
 /// size and a set of points
 pub fn points_to_bytes<const CHUNK_SIZE: usize>(points: &[Fr]) -> Vec<u8> {
     fn point_to_buff<const CHUNK_SIZE: usize>(p: &Fr) -> impl Iterator<Item = u8> {
-        p.into_bigint().to_bytes_le().into_iter().take(CHUNK_SIZE)
+        p.into_bigint().to_bytes_be().into_iter().take(CHUNK_SIZE)
```

Comment:
Shouldn't this also use `fr_to_bytes`? Ok maybe out of scope, but we should have one pass where all these instances are now converted to the utility functions we expose, else once again we can get out of sync.